admin/installer/setup_client.sh

```diff
@@ -55,5 +55,12 @@ ${DOCKER_CERT_HOME}/createclient.exp
 # Copy Docker Certificates directory to Ubuntu's home directory.
 cp -r ${DOCKER_CERT_HOME} ${UBUNTU_HOME}
 
+# Clear out S3 bucket used by CloudFormation so that subsequent stack destroy attempt succeeds.
+s3cmd_wrapper del --recursive --config=/root/.s3cfg s3://${s3_bucket}
```

Comment:
Hmm. Maybe not. The test still passes without changing this and the S3 bucket has been deleted. 

```
(4181)(delete-s3-bucket-cloudformation-flock-4181 ? ▴26)[~/.../HybridLogic/flocker]$ CLOUDFORMATION_TEMPLATE_URL=https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.FLOC-4181.json ACCEPTANCE_YAML=$PWD/acceptance.yml trial admin.test.test_installer
admin.test.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ... Warning: Permanently added '54.183.221.56' (ECDSA) to the list of known hosts.
                                   [OK]

-------------------------------------------------------------------------------
Ran 1 tests in 698.312s

PASSED (successes=1)

```

So I'll merge this once the tests pass.